tests/sles4sap/saptune/mr_test.pm

```diff
@@ -81,16 +81,14 @@ sub test_bsc1152598 {
     my $SLE = is_sle(">=15") ? "SLE15" : "SLE12";
 
     assert_script_run "mr_test verify Pattern/${SLE}/testpattern_bsc1152598#1_1";
-    assert_script_run 'echo -e "[version]\n#\n[block]\nIO_SCHEDULER=none, foobar\n" > /etc/saptune/extra/scheduler-test.conf';
-    # Kernels older than 5.0 use noop
-    script_run '[[ $(uname -r | sed "s/\..*//") -lt 5 ]] && sed -i s/none/noop/ /etc/saptune/extra/scheduler-test.conf';
+    assert_script_run 'echo -e "[version]\n# foobar-NOTE=foobar CATEGORY=foobar VERSION=0 DATE=foobar NAME=\" foobar \"\n[block]\nIO_SCHEDULER=none, foobar\n" > /etc/saptune/extra/scheduler-test.conf';
```

Comment:
What's with all of these `foobar` here and in L90? Isn't there a more descriptive text that can be used?

Reply:
> What's with all of these `foobar` here and in L90? Isn't there a more descriptive text that can be used?

It can be safely ignored.